docs/usage/config-overview.md

```diff
@@ -26,9 +26,10 @@ The default config is loaded first, and may be superseded/overridden by the conf
 ### Global config
 
 Global config means: the config defined by the person or team responsible for running the bot.
-This is also referred to as "bot config", because it's the config passed to the bot by the person running it.
+This is also referred to as "bot config", because it's the config passed to the bot by the administrator that runs the bot.
 Global config can contain config which is "global only" as well as any configuration options which are valid in Inherited config or Repository config.
 
+Global config must be supplied when you run renovate locally or "self-host" renovate.
```

Comment:
```suggestion
Global config must be supplied when you run Renovate locally or "self-host" Renovate.
```

Capitalize proper nouns.

Maybe this sentence needs to be moved down a bit? We mention below how users of the Mend hosted app don't need to worry about global config. But this sentence almost sounds like all users must configure global config.